spec/support/alma_user_stubs.rb

```diff
@@ -0,0 +1,28 @@
+module AlmaUserStubs
+  def stub_alma_facex_user
+    stub_request(:get, "https://api-na.hosted.exlibrisgroup.com/almaws/v1/users/testuser?apikey=#{ENV['ALMA_API_KEY']}&expand=none&user_id_type=all_unique&view=brief").
```

Comment:
Using the `ENV['ALMA_API_BASE_URL']` here and in the other stubbed request would lead to more robust tests over time.

Reply:
done!